app/routes/missions._index.mdx

```diff
@@ -3,38 +3,40 @@ handle:
   breadcrumb: Missions, Instruments, and Facilities
 ---
 
+import { Link } from '@remix-run/react'
+
 import {
   SelectedAlertEmailLink,
   SelectedAlertQuickstartLink,
 } from '~/components/SelectedAlertShortcutLink'
-import { WithFeature } from '~/root'
+import { useFeature } from '~/root'
```

Comment:
According to https://gcn.nasa.gov/docs/contributing/feature-flags, "the `useFeature` method above does not render Markdown that is inside the conditionally displayed text".

Reply:
@jracusin, `useFeature` works fine in MDX documents. It's just that if you do this:

```
Markdown body text {useFeature('FOOBAR') && '[Link text](link target)'}
```

that `[Link text](link target)` will not be processed as Markdown.